packages/server/src/integrations/elasticsearch.ts

```diff
@@ -81,7 +99,25 @@ class ElasticSearchIntegration implements IntegrationBase {
 
   constructor(config: ElasticsearchConfig) {
     this.config = config
-    this.client = new Client({ node: config.url })
+
+    let newConfig = { 
+      node: this.config.url,
+      ssl: this.config.ssl 
+        ? {
+          rejectUnauthorized: this.config.rejectUnauthorized,
+          ca: this.config.ca ? this.config.ca : undefined
```

Comment:
This could be an or statement, e.g. `ca: this.config.ca || undefined`

Reply:
Thanks for the tip! I’ve amended that line.